src/Client/ClientConstants.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Bookboon\OauthClient\Client;
+
+class ClientConstants
```

Comment:
AFAIR, all of the stuff in this folder is here because the OauthClient depends on it. That includes this client constants class. It contains constants that are used by the client. I can move those constants onto another class (maybe the client itself?) But I can't just get rid of the provider or the headers object or the exceptions.

Reply:
JsonLD doesn't have any requirement for this code, so it should be removed. We can also simplify the namespace as mentioned above

Reply:
The Oauth client depends on it. This is the Oauth client repository, not the JsonLD repository. We created this specifically to have a place to put the common Oauth code that was not part of the JsonLD client.